conda_smithy/templates/drone.yml.tmpl

```diff
@@ -10,6 +10,11 @@ platform:
   os: linux
   arch: {{ arch_map[data.build_platform.split("-")[1]] }}
 
+{%- if clone_depth is not none %}
+clone:
+  depth: {{ clone_depth }}
```

Comment:
Zero works on drone too?

Reply:
docs are ambiguous: https://docs.drone.io/pipeline/docker/syntax/cloning/

Reply:
The place where I saw this has turned out to be an archived drone plugin, so I don't know anymore. Is there a good feedstock we can test this on?

Reply:
you can try this one https://github.com/conda-forge/cf-autotick-bot-test-package-feedstock

Reply:
make a PR turning on drone and add this to the ci scripts after rerendering

Reply:
Zero does work on Drone, confirmed: https://cloud.drone.io/conda-forge/cf-autotick-bot-test-package-feedstock/414

Reply:
awesome!